wiki/Beatmap_ranking_procedure/de.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+  - qualification
+  - nomination
+  - nominations
+  - nom
+  - ranking
+  - ranked
+  - Qualifizierung
+  - Nominierung
+  - gerankt
+---
+
+# Ranking-Verfahren für Beatmaps
+
+*Siehe auch: [Rank (Begriffsabgrenzung)](/wiki/Disambiguation/Rank)*
+
+Bevor [Beatmaps](/wiki/Beatmap) in die Kategorie [Ranked](/wiki/Beatmap/Category#ranked) eingestuft werden können, müssen sie von der Community überprüft werden, indem sie das Ranking-Verfahren durchlaufen.
```

Comment:
I had `der Community präsentieren/bereitstellen` at first. This was not enough of a fitting equivalent for me, so I decided to go with this.

Reply:
Note that the original uses `promoted TO the community` (likely in reference to appearing in the ranked category and giving PP), so "von der Community überprüft werden" doesn't seem quite right either...

Reply:
I see. I misunderstood your previous comment. The deviation is fine IMO

Reply:
Sorry that I did not clarify clearly enough that my deviation was on purpose and not a mistake in translation.